src/main/resources/plugin.yml

```diff
@@ -4,11 +4,12 @@ version: 4.9.2
 author: Xanium
 description: A multi-currency economy plugin for spigot servers!
 website: https://www.spigotmc.org/resources/gemseconomy.19655/
-load: STARTUP
 api-version: 1.15
 loadbefore:
   - ItemFrameShops
-softdepend: [Vault, NBTAPI]
+  - Towny
+depend: [NBTAPI]
```

Comment:
GemsEconomy require enable in STARTUP stage to make sure register itself to Vault as fast as possible.

NBTAPI is completely optional.

Reply:
I've undone all the plugin.yml changes.